pycbc/distributions/mass.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (C) 2021 Yifan Wang
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+This modules provides classes for evaluating distributions for mchirp and 
+q (i.e., mass ratio) from uniform component mass.
+"""
+
+import numpy
+from scipy.interpolate import interp1d
+from scipy.special import hyp2f1
+from pycbc.distributions import power_law
+from pycbc.distributions import bounded
+
+class MchirpfromUniformMass1Mass2(power_law.UniformPowerLaw):
+    """ 
+    A distribution for chirp mass from uniform component mass + constraints given by chirp mass.
```

Comment:
This should start on the same line as the `"""`; e.g., `"""A distribution for...`. Also, limit the line width to 79 characters.